SPECS/openssl/openssl.signatures.json

```diff
@@ -1,9 +1,12 @@
 {
  "Signatures": {
-  "ec_curve.c": "138a74284d4193a8eb7c55d3420b9addff81b29687435d6cd1cf8583130e19af",
-  "ectest.c": "a03d49c8de47405912932f946c1f28f2374194cbc053df58dbb228f7ed4b09b3",
-  "hobble-openssl": "7140fcaa43acf82bd40228e0157d42ee8ad52effbffadf84252adab73337d5bd",
-  "ideatest.c": "0ee7bbb2c4a5984015d041135ea645af971cb1a0c7198fc59c7871366fd582f6",
-  "openssl-1.1.1k-hobbled.tar.xz": "ba25a2f2ddaa047d3e07b9b81b9ce7d0af66fce352e7f1608d51c774f2b93f36"
+  "0025-for-tests.patch": "0824fb88488a4e38b5abc6231f0b18b06193444340fa534afc05294f7a13cfcd",
```

Comment:
Our `signatures.json` files typically only list the SourceN files, it's not necessary to list all of the patch files as well